contrib/platform/test/com/sun/jna/platform/win32/COM/IAccessibleTest.java

```diff
@@ -0,0 +1,161 @@
+/* Copyright (c) 2021 Mo Beigi, All Rights Reserved
+ *
+ * The contents of this file is dual-licensed under 2
+ * alternative Open Source/Free licenses: LGPL 2.1 or later and
+ * Apache License 2.0. (starting with JNA version 4.0.0).
+ *
+ * You can freely decide which license you want to apply to
+ * the project.
+ *
+ * You may obtain a copy of the LGPL License at:
+ *
+ * http://www.gnu.org/licenses/licenses.html
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "LGPL2.1".
+ *
+ * You may obtain a copy of the Apache License at:
+ *
+ * http://www.apache.org/licenses/
+ *
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "AL2.0".
+ */
+package com.sun.jna.platform.win32.COM;
+
+import com.sun.jna.platform.win32.Ole32;
+import com.sun.jna.platform.win32.Oleacc;
+import com.sun.jna.platform.win32.User32;
+import com.sun.jna.platform.win32.Variant;
+import com.sun.jna.platform.win32.W32Errors;
+import com.sun.jna.platform.win32.WTypes.BSTRByReference;
+import com.sun.jna.platform.win32.WinDef.HWND;
+import com.sun.jna.platform.win32.WinDef.DWORD;
+import com.sun.jna.platform.win32.WinDef.LONG;
+import com.sun.jna.platform.win32.Guid.REFIID;
+import com.sun.jna.platform.win32.WinNT.HRESULT;
+import com.sun.jna.ptr.LongByReference;
+import com.sun.jna.ptr.PointerByReference;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static com.sun.jna.platform.win32.Oleacc.ROLE_SYSTEM_WINDOW;
+import static com.sun.jna.platform.win32.WinError.S_FALSE;
+import static com.sun.jna.platform.win32.WinError.S_OK;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class IAccessibleTest
+{
+    static {
+        ClassLoader.getSystemClassLoader().setDefaultAssertionStatus(true);
+    }
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        Runtime.getRuntime().exec("calc");
+        Thread.sleep(1000);
+
+        // Initialize COM for this thread...
+        HRESULT hr = Ole32.INSTANCE.CoInitialize(null);
+
+        if (W32Errors.FAILED(hr)) {
+            tearDown();
+            throw new COMException("CoInitialize() failed");
+        }
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        Runtime.getRuntime().exec("taskkill.exe /f /im calculator.exe");
+        Thread.sleep(1000);
+        Ole32.INSTANCE.CoUninitialize();
+    }
+
+    private static HWND getCalculatorHwnd() {
```

Comment:
These two private helpers are duped in both test files because I didn't want to have them public or introduce any sort of cross dependency.